net-misc/icingadb/icingadb-1.2.0.ebuild

```
@@ -0,0 +1,57 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module systemd
+
+DESCRIPTION="Icinga configuration and state database supporting multiple environments"
+HOMEPAGE="https://icinga.com/docs/icinga-db/"
+SRC_URI="
+	https://github.com/Icinga/icingadb/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	https://dev.gentoo.org/~sbraz/distfiles/${P}-deps.tar.xz
+"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64"
+IUSE="migrate"
+
+RDEPEND="
+	acct-user/icinga
+	acct-group/icinga
+"
+
+DOCS=( config.example.yml doc README.md schema )
+
+src_prepare() {
+	default
+	sed -e "s|@EPREFIX@|${EPREFIX}|" "${FILESDIR}/icingadb.service" > "${WORKDIR}/icingadb.service" || die
```

```suggestion
	sed -e "s|@EPREFIX@|${EPREFIX}|" "${FILESDIR}/icingadb.service" > "${T}/icingadb.service" || die
```

`${T}` is probably the more "legal" place to use here, but I don't think `${WORKDIR}` is **wrong** either. (if changed, needs to be updated in `src_install` too).